azuredeploy.json

```diff
@@ -0,0 +1,68 @@
+{
+    "parameters": {
+        "siteName": {
+            "type": "string"
+        },
+        "servicePlanName": {
+            "type": "string"
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
+            "allowedValues": [
+                "West US",
+                "West Europe",
+                "Southeast Asia"
+            ]
+        },
+        "sku": {
+            "type": "string",
+            "defaultValue": "S3",
+            "allowedValues": [
+                "B2",
+                "B3",
+                "S2",
+                "S3"
+            ]
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Web/sites",
+            "name": "[parameters('siteName')]",
+            "properties": {
+                "siteConfig": {
+                    "appCommandLine": "",
+                    "linuxFxVersion": "DOCKER|fanjeffrey/python"
```

Comment:
After I build the official image, you will have to update the ARM template to use that image instead. Something like appsvc/apps:python-0